### ISSUE TITLE
Absence of fetal nasal bone

### ISSUE THREAD

**Issue by BimalChaudhari** [obophenotype/human-phenotype-ontology#6988]
**Preferred term label:**
Absence of fetal nasal bone

**Synonyms**
Absent nasal bone

**Definition (free text, please give PubMed ID)**
The nasal bone is considered absent when it is not visualized on a midsagittal view of the profile.

In the second trimester, a true midsagittal view of the fetal profile is obtained and magnified to fill the majority of the image space. The nasal bone appears as an echogenic linear structure below the skin edge. The optimal angle of insonation is 45 degrees to the longitudinal axis of the fetal nasal bone. If the angle of insonation is 0 or 180 degrees, the nasal bone may appear artificially absent. The presence or absence of the nasal bone may be determined at the time of the 11- to 14-week ultrasound examination and used as part of the risk assessment for aneuploidy

https://doi.org/10.1016/j.ajog.2019.08.049  (PMID: 31679594)

**Parent term (use hpo.jax.org/app)**
I propose a new parent term abnormality of fetal nasal bone

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 190685 
ORPHA:3378
ORPHA:3380
ORPHA:250
**Your nano-attribution (ORCID)**
0000-0002-0115-949X

**Comment by patt1morgan** [comment]
@Shagun1976, @fcradio, @Green-eve.   As you can see, here is another request for nasal bone term.  Thanks in advance for reviewing it.

**Comment by pnrobinson** [comment]
added
